packages/mui-base/src/SliderUnstyled/SliderValueLabelUnstyled.tsx

```diff
@@ -43,6 +46,8 @@ export default function SliderValueLabelUnstyled(props) {
 SliderValueLabelUnstyled.propTypes = {
   children: PropTypes.element.isRequired,
   className: PropTypes.string,
+  open: PropTypes.bool,
+  // eslint-disable-next-line react/no-unused-prop-types
   theme: PropTypes.any,
```

Comment:
I think we can remove `theme` prop type validation because it is not being used.